MobileFoodSchedules/MobileFoodSchedules/Models/MobileFoodPermit.cs

```diff
@@ -17,89 +9,117 @@ namespace QuickType
 
     public partial class MobileFoodPermit
     {
+        private long objectid;
+        private string applicant;
+        private Facilitytype? facilitytype;
+        private long cnn;
+        private string locationdescription;
+        private string address;
+        private string blocklot;
+        private string block;
+        private string lot;
+        private DateTimeOffset? noisent;
+        private string dayshours;
+        private Location location;
+        private DateTimeOffset? expirationdate;
+        private long priorpermit;
+        private DateTimeOffset received;
+        private DateTimeOffset? approved;
+        private Uri schedule;
+        private string longitude;
+        private string latitude;
+        private string y;
+        private string x;
+        private string fooditems;
+        private Status status;
+        private string permit;
+
         [JsonProperty("objectid")]
         [JsonConverter(typeof(ParseStringConverter))]
-        public long Objectid { get; set; }
+        public long Objectid { get => objectid; set => objectid = value; }
```

Comment:
Encapsulation is one of the four components of object oriented design.  I'm good either way here, because .net often considers { get; set; } to be a shortcut for encapsulation.